api/envoy/config/core/v3/health_check.proto

```diff
@@ -80,11 +80,22 @@ message HealthCheck {
     }
   }
 
-  // [#next-free-field: 13]
+  // [#next-free-field: 14]
   message HttpHealthCheck {
     option (udpa.annotations.versioning).previous_message_type =
         "envoy.api.v2.core.HealthCheck.HttpHealthCheck";
 
+    enum Method {
```

Comment:
You can already use https://github.com/envoyproxy/envoy/blob/b75e0b51e017bb45ccc8db9f144d63dcccf05f4d/api/envoy/config/core/v3/base.proto#L44